src/components/posts/UserPostListItem.js

```diff
@@ -23,6 +24,7 @@ export const UserPostListItem = props => {
       <div className="userPostListItem--col-right">
         <p className="userPostListItem__category">{category.name}</p>
         <ConfirmableDeleteButton onDelete={() => handleDelete(id)} />
+        <EditPostButton action={props} postId={id} />
```

Comment:
There's really no reason to pass anything down in this `action` prop since we're never using it, so `action` could just be deleted here for slight cleanup.